topi/include/topi/elemwise.h

```diff
@@ -360,5 +359,71 @@ inline Tensor full_like(const Tensor& x,
   }, name, tag);
 }
 
+ /*
+ * \brief Fast exponential function implementation from Eigen
+ * https://github.com/eigenteam/eigen-git-mirror/blob/master/Eigen/src/Core/arch/Default/GenericPacketMathFunctions.h#L183
```

Comment:
What is the license of libeigen? If you understand and write the code by yourself, could remove it.

Reply:
@FrozenGene 
Eigen licence is MPL2 https://www.mozilla.org/en-US/MPL/2.0/
I wrote this fastexp implementation and understand the code.  Eigen github link in the comment is for algorithm reference.

Reply:
does MPL2 allow somebody modify the code and doesn't open source? This is critical for company using TVM. I worried about it. However, I am not the expert of open source license. Maybe @tqchen could have more authoritative answer about it. My previous meaning is if you understand it and write it by yourself, you could remove this link so that we could avoid license problem.

Reply:
The code in the main repo need to be licensed under ASv2.

If the code comes from a difference license, We will need to put the code in the thirdparty and specify the license clearly. If we reference an existing algorithm and is implemented from scratch. It is better to declare ASv2

Reply:
Please see updated fast_exp implementation.